ccd-definition/CaseEvent/User/UserEvents-SDO-non-prod.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
```

Comment:
Pretty sure this file name should be appended with "nonprod" rather than "non-prod" as from memory the script to build the different configs looks for nonprod.

See similar files here:
https://github.com/hmcts/civil-ccd-definition/tree/master/ccd-definition/CaseEventToComplexTypes